### ISSUE TITLE
angular-markdown-directive is abandoned

### ISSUE THREAD

**Issue by rpkarlsson** [Kodkollektivet/kodkollektivet.se-old#10]
As the header says A-M-D seems abandoned. No updates since 5 Dec 2015 and 12 unmerged pull requests. Further more installing the correct showdown versions triggers a deprecated warning.

https://github.com/btford/angular-markdown-directive

Can we move away from this dependency somehow? 

Perhaps: https://github.com/Hypercubed/angular-marked could work as a suitable replacement as it is based on A-M-D, seems to match functionality, expand functionality and be better updated?

**Comment by jherrlin** [comment]
I don’t see any reason to stay with an outdated lib,
As you say angular-marked looks good! 
I guess its easy to move away from angular-markdown-directive, directives that depends on angular-markdown-directive is in project details and contributor details.

**Comment by jherrlin** [comment]
This one is resolved, right? @rk222ev

**Comment by rpkarlsson** [comment]
I dont think so. I think were still depending on angular-markdown-directive/btford.markdown.
